.github/workflows/coverage.yml

```diff
@@ -32,13 +32,13 @@ jobs:
         run: yarn test
 
       - name: Create Coverage Badges
-        uses: jaywcjlove/coverage-badges-cli@e07f25709cd25486855c1ba1b26da53576ff3620
+        uses: jaywcjlove/coverage-badges-cli@f13e40c05984776612eb7ee8878a5e81233d7313 # v1.1.0
         with:
           source: coverage/coverage-summary.json
           output: coverage/lcov-report/badges.svg
 
       - name: Deploy
-        uses: JamesIves/github-pages-deploy-action@8817a56e5bfec6e2b08345c81f4d422db53a2cdc
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
Use commit SHA please

Reply:
I tried the latest commit `cd53a4b70a96f186f4f4ba6aa288f80e03cc8757` from the default branch (`dev`) but I got the following error:
```Error: File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/cd53a4b70a96f186f4f4ba6aa288f80e03cc8757/lib/main.js'```

 Let me search for the latest commit

Reply:
Done. Could you re-review, please? @ilfa